webclient/app/src/app/commons/alert/ConfirmationDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { useTranslation } from 'react-i18next';
+
+export default function ConfirmationDialog(props) {
+    const { content, open, onClose, onSubmit } = props;
```

Comment:
Hier ist es aber generell eine gute Idee content in title und message aufzuspalten. Für den Entwickler ist es dann leichter,  diese Komponente zu nutzen